conda_env/__init__.py

```diff
@@ -1,4 +0,0 @@
-# Copyright (C) 2012 Anaconda, Inc
-# SPDX-License-Identifier: BSD-3-Clause
-from conda import __version__
-__version__ = __version__
```

Comment:
Should this file get a PendingDeprecation warning in case someone is looking for `conda_env.__version__`, until the point in the cycle where it can be removed?

Reply:
Like... conda? https://github.com/conda/conda/blob/ed75abca38ae05b225fe36774eadd295d2c52eb8/conda/cli/main_info.py#L123

Reply:
good find, this value is unused in `conda info` but is still included in `conda info --json`